scripts/set-env.sh

```diff
@@ -0,0 +1,8 @@
+export PROJECT=$(gcloud config get-value project)
+export ZONE=$(gcloud config get-value compute/zone)
+export GOVERNANCE_PROJECT=$(gcloud config get-value project)
+export WINDOWS_CLUSTER=false
+export PUBLIC_CLUSTER=false
+export STATE=gcs
```

Comment:
I would recommend defaulting to local state. Can you add a small section to the appropriate readme ( for now) on how to leverage the Remote State functionality?

Reply:
updated